### PR TITLE
Add libsdformat 6.2.0

### PR DESCRIPTION
**Pull request by seanyen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
cc @wolfv @traversaro @j-rivero this is another gazebo deps. Leave a comment below if you are interested to be a co-maintainer.

File: recipes/sdformat/bld.bat [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON ^
+    -DBOOST_ROOT=%LIBRARY_PREFIX% ^
+    -DBoost_NO_SYSTEM_PATHS=ON ^
+    -DBoost_NO_BOOST_CMAKE=ON ^
+    -DBoost_DEBUG=ON ^
+    -DUSE_EXTERNAL_TINYXML=ON ^
+    -DUSE_INTERNAL_URDF=OFF ^
+    -DURDF_FOUND=ON ^
```
Comment (seanyen) [inline recipes/sdformat/bld.bat]: I created a TODO for `urdfdom-feedstock`: https://github.com/conda-forge/urdfdom-feedstock/issues/2 :)

**Comment by wolfv** [comment]
Sure, happy to be a maintainer. We should ask to make a ROS team, so we can just add that team and everyone gets the correct maintainer rights.

File: recipes/sdformat/build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON \
+      -DBOOST_ROOT=$PREFIX \
+      -DBoost_NO_SYSTEM_PATHS=ON \
+      -DBoost_NO_BOOST_CMAKE=ON \
+      -DBoost_DEBUG=ON \
+      -DUSE_EXTERNAL_TINYXML=ON \
+      -DUSE_INTERNAL_URDF=OFF \
+      -DCMAKE_VERBOSE_MAKEFILE=ON
+
+cmake --build . --config Release
```
Comment (isuruf) [inline recipes/sdformat/build.sh]:
```suggestion
cmake --build . --config Release -- -j$CPU_COUNT
```

**Comment by seanyen** [comment]
cc @conda-forge/staged-recipes this is ready for review and merge.

**Comment by wolfv** [comment]
@seanyen i just asked regarding a conda-forge team for the ROS maintainers and they said that using an existing feedstocks team would be a good solution. 
We could use the https://github.com/conda-forge/ros-core-feedstock or the https://github.com/conda-forge/ros-catkin-feedstock as the ros team? what do you think?

then you could just add @conda-forge/ros-core to the maintainers and it should give maintainer rights to you, me and @lesteve.

**Comment by seanyen** [comment]
> then you could just add @conda-forge/ros-core to the maintainers and it should give maintainer rights to you, me and @lesteve.

Sure, let me add the team.

**Comment by lesteve** [comment]
> then you could just add @conda-forge/ros-core to the maintainers and it should give maintainer rights to you, me and @lesteve.

Fine to be added to @conda-forge/ros-core. Since I know very little about ROS, I'll try to get some Robotics people around me involved there as well.

**Comment by seanyen** [comment]
> * do you have ambitious goals like @wolfv and want to have a minimal ROS distribution inside-conda-forge with Windows support?

In earlier this year, my team managed to bring up ROS Melodic on Windows, up to `desktop_full` (https://aka.ms/ros). In order to expand & motivate more ROS packages to be ported on Windows from the community, I am exploring a model to let everyone who is interested to contribute. With the great work done by @wolfv, it should be much easier to bring up the minimal ROS on Windows from `conda-forge`, and that's what I'd love to see too. For now, I am focused on the lower layer dependencies bring-up (e.g. Gazebo9), which is for my owned evaluation. :)

**Comment by j-rivero** [comment]
> cc @wolfv @traversaro @j-rivero this is another gazebo deps. Leave a comment below if you are interested to be a co-maintainer.

Sure. Feel free to add me as co-maintainer for Gazebo & friends. Thanks Sean!

**Comment by seanyen** [comment]
cc @conda-forge/staged-recipes this is ready for review and merge.

**Pull request merged**